api/src/opentrons/protocol_engine/commands/temperature_module/wait_for_temperature.py

```diff
@@ -57,23 +59,25 @@ async def execute(self, params: AwaitTemperatureParams) -> AwaitTemperatureResul
             await temp_hardware_module.await_temperature(
                 awaiting_temperature=target_temp
             )
-        return AwaitTemperatureResult()
+        return WaitForTemperatureResult()
 
 
-class AwaitTemperature(BaseCommand[AwaitTemperatureParams, AwaitTemperatureResult]):
+class WaitForTemperature(
+    BaseCommand[WaitForTemperatureParams, WaitForTemperatureResult]
+):
     """A command to set a Temperature Module's target temperature."""
```

Comment:
Just noticed this docstring

```suggestion
    """A command to wait for a Temperature Module's target temperature."""
```